functions/delete-all/1.1/index.js

```diff
@@ -0,0 +1,76 @@
+import templayed from '../../utils/templayed';
+
+const getAll = async (modelName, where, queryBody) => {
+  const queryName = `all${modelName}`;
+  const getAllQuery = `
+    query {
+      ${queryName}(where: ${where}, take: 200) {
+        ${queryBody}
+      }
+    }
+  `;
+
+  const { data, errors } = await gql(getAllQuery);
```

Comment:
please use variables like: https://github.com/bettyblocks/cli/wiki/Functions:DataAPI#read

Reply:
Fixed. The keys in the string of the variable "where" needed double quotes in order to parse it to a JSON object since `JSON.parse('{ id: { gt: 100 } }')` doesn't work. It had to be converted to `JSON.parse('{ "id": { "gt": 100 } }')` and that is what the regex placer `.replace(/(['"])?([a-z0-9A-Z_]+)(['"])?:/g, '"$2": ')` does.